accounts/forms.py

```diff
@@ -345,8 +352,31 @@ class EmailSettingsForm(forms.Form):
     )
 
 
-class FsPasswordResetForm(PasswordResetForm):
-    def get_users(self, email):
+class FsPasswordResetForm(forms.Form):
```

Comment:
Isn't there a way that we could reuse more from Django's `PasswordResetForm`?
For example, the `send_email` and `save` methods look the same after a quick inspection.
Why do we need to create the form "from scratch"?

Reply:
After talking with @alastair he clarified that you tried to reuse as much as possible but had to reimplement it anyway. I'll add a comment to the code and an entry to the [Migration to a new version of Django](https://github.com/MTG/freesound/wiki/Migration-to-a-new-version-of-Django) wiki doc to check for changes in this code when upgrading.